### ISSUE TITLE
1.20 branch on spigot error

### ISSUE THREAD

**Issue by sebseb7** [FancyMcPlugins/FancyHolograms#14]
commit: cb6852e26a765608e480b64a168f42242ecf395c

```

[20:19:22] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to FancyHolograms v1.1.0
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:528) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.level.PlayerInteractManager.a(PlayerInteractManager.java:503) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1637) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.network.protocol.game.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.network.protocol.game.PacketPlayInUseItem.a(PacketPlayInUseItem.java:1) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:31) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandler.d(SourceFile:156) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(SourceFile:23) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1152) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:1) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandler.x(SourceFile:130) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.bg(MinecraftServer.java:1131) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1124) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:139) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.p_(MinecraftServer.java:1108) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1019) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NoSuchMethodError: 'boolean org.bukkit.persistence.PersistentDataContainer.has(org.bukkit.NamespacedKey)'
        at de.oliver.fancylib.itemClick.PlayerInteractListener.onPlayerInteract(PlayerInteractListener.java:16) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        ... 22 more

```

and

```
org.bukkit.command.CommandException: Unhandled exception executing command 'hologram' in plugin FancyHolograms v1.1.0
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R1.CraftServer.dispatchCommand(CraftServer.java:864) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at org.bukkit.craftbukkit.v1_20_R1.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:50) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:265) ~[spigot-1.20-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:314) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:298) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1957) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.network.PlayerConnection.lambda$18(PlayerConnection.java:1919) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandler.b(SourceFile:67) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandler.d(SourceFile:156) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(SourceFile:23) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1152) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:1) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandler.x(SourceFile:130) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.bg(MinecraftServer.java:1131) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1124) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:139) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.p_(MinecraftServer.java:1108) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1019) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3777-Spigot-723951c-f3b2b22]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NoClassDefFoundError: net/kyori/adventure/audience/Audience
        at de.oliver.fancyholograms.commands.HologramCMD.create(HologramCMD.java:576) ~[?:?]
        at de.oliver.fancyholograms.commands.HologramCMD.onCommand(HologramCMD.java:157) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        ... 23 more
Caused by: java.lang.ClassNotFoundException: net.kyori.adventure.audience.Audience
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:147) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:99) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at de.oliver.fancyholograms.commands.HologramCMD.create(HologramCMD.java:576) ~[?:?]
        at de.oliver.fancyholograms.commands.HologramCMD.onCommand(HologramCMD.java:157) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
        ... 23 more
```

**Comment by OliverSchlueter** [comment]
Try with paper when it's available

**Comment by sebseb7** [comment]
a dev build of paper is out and there is issues ...

```
[02:28:29 INFO]: This server is running Paper version git-Paper-"318a08c" (MC: 1.20) (Implementing API version 1.20-R0.1-SNAPSHOT) (Git: 318a08c on dev/1.20)
```



```

[02:28:50 INFO]: seb issued server command: /fancyholograms
[02:29:06 INFO]: seb issued server command: /hologram create info
[02:29:06 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'hologram' in plugin FancyHolograms v1.1.0
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.20-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R1.CraftServer.dispatchCommand(CraftServer.java:929) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at org.bukkit.craftbukkit.v1_20_R1.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:265) ~[paper-1.20.jar:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:324) ~[?:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:308) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.performChatCommand(ServerGamePacketListenerImpl.java:2295) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$20(ServerGamePacketListenerImpl.java:2255) ~[?:?]
        at net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:59) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1341) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1318) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1311) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1289) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1177) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.20.jar:git-Paper-"318a08c"]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalAccessError: class de.oliver.fancyholograms.Hologram tried to access private field net.minecraft.world.entity.Entity.t (de.oliver.fancyholograms.Hologram is in unnamed module of loader 'FancyHolograms-1.1.0.ja
r' @2c46244e; net.minecraft.world.entity.Entity is in unnamed module of loader java.net.URLClassLoader @49097b5d)
        at de.oliver.fancyholograms.Hologram.updateLocation(Hologram.java:126) ~[FancyHolograms-1.1.0.jar:?]
        at de.oliver.fancyholograms.Hologram.spawn(Hologram.java:100) ~[FancyHolograms-1.1.0.jar:?]
        at de.oliver.fancyholograms.commands.HologramCMD.create(HologramCMD.java:588) ~[FancyHolograms-1.1.0.jar:?]
        at de.oliver.fancyholograms.commands.HologramCMD.onCommand(HologramCMD.java:157) ~[FancyHolograms-1.1.0.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.20-R0.1-SNAPSHOT.jar:?]
        ... 23 more
[02:29:37 INFO]: seb issued server command: /hologram create
[02:29:39 ERROR]: Could not pass event PlayerMoveEvent to FancyHolograms v1.1.0
java.lang.IllegalAccessError: class de.oliver.fancyholograms.Hologram tried to access private field net.minecraft.world.entity.Entity.t (de.oliver.fancyholograms.Hologram is in unnamed module of loader 'FancyHolograms-1.1.0.jar' @2c46244
e; net.minecraft.world.entity.Entity is in unnamed module of loader java.net.URLClassLoader @49097b5d)
        at de.oliver.fancyholograms.Hologram.updateLocation(Hologram.java:126) ~[FancyHolograms-1.1.0.jar:?]
        at de.oliver.fancyholograms.Hologram.spawn(Hologram.java:100) ~[FancyHolograms-1.1.0.jar:?]
        at de.oliver.fancyholograms.listeners.PlayerMoveListener.onPlayerMove(PlayerMoveListener.java:37) ~[FancyHolograms-1.1.0.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor5.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.20-R0.1-SNAPSHOT.jar:?]
        at co
```

**Comment by OliverSchlueter** [comment]
Fixed in 5e0f49f9bcb238d1bd06c73194e71a16a4472181
